dotcom-rendering/src/web/components/SignInGate/gates/main-variant.tsx

```diff
@@ -15,19 +15,57 @@ const SignInGateMain = React.lazy(() => {
 	});
 });
 
+const SignInGateMainCheckoutComplete = React.lazy(() => {
+	const { start, end } = initPerf('SignInGateMainCheckoutComplete');
+	start();
+	return import(
+		/* webpackChunkName: "SignInGateMain" */ '../gateDesigns/SignInGateMainCheckoutComplete'
+	).then((module) => {
+		end();
+		return { default: module.SignInGateMainCheckoutComplete };
+	});
+});
+
+// TODO investigate data tracking component name requirements?
+// TODO Add context Comments
+const showCheckoutCompletePersonalisation = true;
+
 export const signInGateComponent: SignInGateComponent = {
-	gate: ({ ophanComponentId, dismissGate, guUrl, signInUrl, abTest }) => (
-		<Lazy margin={300}>
-			<Suspense fallback={<></>}>
-				<SignInGateMain
-					ophanComponentId={ophanComponentId}
-					dismissGate={dismissGate}
-					guUrl={guUrl}
-					signInUrl={signInUrl}
-					abTest={abTest}
-				/>
-			</Suspense>
-		</Lazy>
-	),
+	gate: ({
+		ophanComponentId,
+		dismissGate,
+		guUrl,
+		signInUrl,
+		abTest,
+		checkoutCompleteCookieData,
+	}) => {
+		return (
+			<Lazy margin={300}>
+				<Suspense fallback={<></>}>
+					{showCheckoutCompletePersonalisation &&
+					checkoutCompleteCookieData !== undefined ? (
```

Comment:
I think it is more idiomatic how it was previously and so you don't get into have to think about double negatives. 
```
return undefined ? "a" : "b" --> "b"
```

Reply:
yeah this is more idiomatic. I've been tripped up by the `truthiness` of JS before, so I like to be explicit, but perhaps that is overkill